csv/src/fs2/data/csv/internals/RowParser.scala

```diff
@@ -18,11 +18,11 @@ package data
 package csv
 package internals
 
-import cats.data.{State => _, _}
-
 private[csv] object RowParser {
 
-  def pipe[F[_]](separator: Char)(implicit F: RaiseThrowable[F]): Pipe[F, Char, NonEmptyList[String]] = {
+  def pipe[F[_]](separator: Char,
+                 quoteHandling: QuoteHandling = QuoteHandling.Adaptive)(
```

Comment:
I guess `Adaptive` is a renaming artefact and should be `QuoteHandling.RFCCompliant`?

Reply:
This is what happens when I try to refactor between meetings. Fixing these up. But yes, renamed.